test/specs/ExpectNoAttributeValueSpec.ts

```diff
@@ -0,0 +1,31 @@
+import { assert, expect } from "chai";
+import { BrowserUtils } from "../../src/commons/BrowserUtils";
+
+// tslint:disable-next-line:no-http-string
+const sampleAppUrl: string = "http://127.0.0.1:8000/";
+
+/**
+ * wdio-allure-ts navigateToUrl action test
+ */
+describe("ExpectNoAttributeValueSpec of BrowserUtils Tests", () => {
+    it("Validate positive result ", () => {
+        BrowserUtils.navigateToUrl(sampleAppUrl);
+        assert.equal(BrowserUtils.getAttribute("//form",'method'), "post");
```

Comment:
It's nice to have test for getAttribute api (move it to GetAttributeSpec.ts) but you change was related to expectNoAttributeValue so you need to add test for it

Reply:
done